processing/src/main/java/org/apache/druid/segment/data/SingleIndexedInt.java

```diff
@@ -33,6 +33,11 @@ public SingleIndexedInt()
   {
   }
 
+  public SingleIndexedInt(int value)
```

Comment:
I believe one of the reasons for not having this constr was to avoid people creating new objects and reusing this class in the selectors, by caching an instance and calling setValue() every time getObject is called from selector. 
Can we do the same and use setValue instead of adding new constr ?

Reply:
Thanks. Fixed.